### PR TITLE
Changes to build on OSX Yosemite.

### PR DESCRIPTION
**Pull request by gartung**

### CODE REVIEW

File: apt.spec [hunk 1]
```diff
@@ -4,7 +4,8 @@
 ## INITENV CMD_CSH if ( -f %{instroot}/common/apt-site-env.csh )  source %{instroot}/common/apt-site-env.csh; endif
 ## NOCOMPILER
 
-Source0: git://github.com/cms-sw/apt-rpm.git?obj=master/%{realversion}&export=apt-rpm-%{realversion}&output=/apt-rpm-%{realversion}.tgz
+Source0: https://github.com/cms-sw/apt-rpm/archive/%{realversion}.tar.gz
```
Comment (ktf) [inline apt.spec:7]: why moving to the tarball?

File: apt.spec [hunk 2]
```diff
@@ -4,7 +4,8 @@
 ## INITENV CMD_CSH if ( -f %{instroot}/common/apt-site-env.csh )  source %{instroot}/common/apt-site-env.csh; endif
 ## NOCOMPILER
 
-Source0: git://github.com/cms-sw/apt-rpm.git?obj=master/%{realversion}&export=apt-rpm-%{realversion}&output=/apt-rpm-%{realversion}.tgz
```
Comment (gartung) [inline apt.spec:7]: This url did not seem to work.

File: apt.spec (hunk 1 above)
Comment (gartung) [inline apt.spec:7]: See comments inline.

**Comment by gartung** [comment]
Also see http://home.fnal.gov/~gartung/scan-build/build-rpm-osx.sh. I needed to add the flags --use-system-compiler --skip-pre-install-checks  to the cmsBuild command for bootstrap-driver.

File: apt.spec (hunk 1 above)
Comment (gartung) [inline apt.spec:7]: I was having problems pulling with the git url.

File: rpm-4.8.0-htonll-fix.patch [hunk 3]
```diff
@@ -0,0 +1,48 @@
+diff --git a/lib/header.c b/lib/header.c
```
Comment (ktf) [inline rpm-4.8.0-htonll-fix.patch:1]: I created https://github.com/cms-externals/rpm which has all the current patches but this one in the `cms/v4.8.0` branch. Does this one work on slc6?

Comment (gartung) [inline rpm-4.8.0-htonll-fix.patch:1]: Yes.

Sent on the new Sprint Network from my Samsung Galaxy S®4.

-------- Original message --------
From: Giulio Eulisse
Date:05/16/2015 2:22 AM (GMT-06:00)
To: cms-sw/cmsdist
Cc: Patrick E Gartung
Subject: Re: [cmsdist] Changes to build on OSX Yosemite. (#1558)

In rpm-4.8.0-htonll-fix.patchhttps://github.com/cms-sw/cmsdist/pull/1558#discussion_r30459217:

> @@ -0,0 +1,48 @@
> +diff --git a/lib/header.c b/lib/header.c

I created https://github.com/cms-externals/rpm which has all the current patches but this one in the cms/v4.8.0 branch. Does this one work on slc6?

Comment (ktf) [inline rpm-4.8.0-htonll-fix.patch:1]: Ok, perfect. I've added your patch to https://github.com/cms-externals/rpm/tree/cms/v4.8.0. Will soon adapt this PR to pick up sources from there.

**Pull request closed without merging**